Week 6+/The Movie App/app/src/main/java/codes/jenn/movieapp/di/ApplicationModule.kt

```diff
@@ -0,0 +1,34 @@
+package codes.jenn.movieapp.di
+
+import android.content.Context
+import androidx.work.WorkManager
+import codes.jenn.movieapp.common.utils.CredentialsValidator
+import codes.jenn.movieapp.common.utils.Validator
+import codes.jenn.movieapp.common.utils.WorkManagerHelper
+import codes.jenn.movieapp.login.view.LoginActivity
+import codes.jenn.movieapp.prefs.SharedPrefsManager
+import codes.jenn.movieapp.worker.SynchronizeMoviesWorker
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module
+
+val applicationModule = module {
+  single {
+    SharedPrefsManager(
+      get(),
+      androidContext().getSharedPreferences(MOVIES_SHARED_PREFS, Context.MODE_PRIVATE)
```

Comment:
I would also provide the prefs beforehand so that we can just use get here.

Reply:
Ah, good point.
Kind of missed that. Thanks